src/ucp/core/ucp_ep.h

```diff
@@ -60,7 +60,7 @@ enum {
     UCP_EP_FLAG_CLOSE_REQ_VALID        = UCS_BIT(11),/* close protocol is started and
                                                         close_req is valid */
     UCP_EP_FLAG_ERR_HANDLER_INVOKED    = UCS_BIT(12),/* error handler was called */
-    UCP_EP_FLAG_TEMPORARY              = UCS_BIT(13),/* the temporary EP which holds
+    UCP_EP_FLAG_INTERNAL               = UCS_BIT(13),/* the temporary EP which holds
```

Comment:
is the comment still correct for memtype EPs?

Reply:
fixed